src/prommis/uky/uky_flowsheet.py

```diff
@@ -1357,6 +1428,11 @@ def initialize_system(m):
         m.fs.solex_cleaner_strip,
     ]
 
+    initializer_precip = OxalatePrecipitatorInitializer()
+    precip_units = [
+        m.fs.precipitator,
+    ]
+
```

Comment:
Running into the following issue with OxalatePrecipitatorInitializer: `idaes.core.util.exceptions.InitializationError: Degrees of freedom for fs.precipitator.mscontactor were not equal to zero during initialization (DoF = 13).` The SXInitializer had a similar issue when it was first being developed - see Andrew's comment here on a potential fix: https://github.com/prommis/prommis/pull/85#issuecomment-2383555218